src/lumen/activity/field.cpp

```diff
@@ -0,0 +1,64 @@
+#include "lumen/activity/field.hpp"
+
+namespace lumen::activity {
+
+// Field implementation.
+template <typename T>
+void Field<T>::signal_update()
+{
+    if (parent_ != nullptr) {
+        parent_->update_display();
+    }
+}
+
+// NumberField implementation.
+NumberField::NumberField(Activity* parent, uint increment /* = 1 */)
+    : Field{parent, {}}, increment_{increment}
```

Comment:
- [ ] Set the default value of `value_` to be 0

Reply:
`{}` initializes the object with it's default value, which in this case is 0.